backend/de.metas.adempiere.adempiere/base/src/test/java/de/metas/impexp/spreadsheet/csv/CSVWriterTest.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.impexp.spreadsheet.csv;
+
+import com.google.common.collect.ImmutableList;
+import de.metas.i18n.Language;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import static de.metas.impexp.spreadsheet.csv.CSVWriter.DEFAULT_FieldDelimiter;
+
+public class CSVWriterTest
+{
+	private static final String FILE_NAME = "./src/test/resources/de/metas/impexp/spreadsheet/csv/test.csv";
+	private static final List<String> HEADER = ImmutableList.of("id", "name");
+
+	private File outputFile;
+	private Path path;
+
+	@BeforeEach
+	public void init() throws IOException
+	{
+		cleanUpFiles();
+
+		outputFile = new File(FILE_NAME);
+		outputFile.createNewFile();
+
+		path = Paths.get(FILE_NAME);
+	}
+
+	@AfterEach
+	public void cleanUpFiles()
+	{
+		final File targetFile = new File(FILE_NAME);
+		targetFile.delete();
+	}
+
+	@Test
+	public void givenDoNotQuoteRowsAndNoDelimiterInText_whenAppendRows_thenNoQuotesApplied() throws IOException
+	{
+		final List<Object> row1 = ImmutableList.of("firstRowFirstColumn", "firstRowSecondColumn");
+		final List<Object> row2 = ImmutableList.of("secondRowFirstColumn", "secondRowSecondColumn");
+
+		final CSVWriter csvWriter = CSVWriter.builder()
+				.outputFile(outputFile)
+				.header(HEADER)
+				.adLanguage(Language.AD_Language_en_US)
+				.fieldDelimiter(DEFAULT_FieldDelimiter)
```

Comment:
minor: pls inline the delimiter... there is no point to have it as a constant since your expectation is not using it